reference/glossary.rst

```diff
@@ -810,13 +810,24 @@ Glossary
         *Work in Progress*
 
     jitter
-        *Work in Progress*
+        Jitter is the variation in delay or latency between when data packets are sent and when they are received over a network, causing irregular arrival times at the destination. This variation is often caused by network congestion, packet loss, poor hardware performance or differences in the path packets take. 
+
+        Related topic(s): :term:`Chrony`.
 
     journald
-        *Work in Progress*
+        Journald is a logging driver that sends container logs to the systemd journal. 
+
+        See `journald <https://docs.docker.com/engine/logging/drivers/journald/>`_ for more details.
+
+        Related topic(s): :term:`Docker`.
```

Comment:
I think we can put "logging" here as a topic as well as Docker :)

Reply:
(which has made me realise that we only have journald content under Docker right now, which we'll need to queue up as follow-up improvements!)